### PR TITLE
[New Model] Electric drive passport 1.0.0

### LINKED ISSUE #532: [New Model]: E-Drive
## Model Description
E-Drive passport as a deviation from the Digital Product Passport.
Includes attributes from the transmission.transmission_passport, which will be deprecated in the future.

## Example Data

- DPP general data for metadata of a passport
- Transmission data

## MS1 Criteria

- [x] The proposed aspect model does not exist already
- [x] The proposed aspect model does not extend an existing aspect but introduces completely new functionality
- [x] The proposal references a Catena-X use case
- [x] Relevant standards are mentioned/linked
- [x] A first draft of the data structure is provided in an example
- [x] A modeling steward has been assigned

### PR DESCRIPTION
**Pull request by catroest**
## Description

 -->

Closes #532 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.5.1)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by catroest** [comment]
@agg3fe ready for review

File: io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+   samm-c:baseCharacteristic :StringList ;
+   samm-c:constraint :VehicleTypesConstraint .
+
+:ServiceCharacteristic a samm:Characteristic ;
+   samm:preferredName "Service Characteristic"@en ;
+   samm:description "Characteristic for the service dates."@en ;
+   samm:dataType :ServiceEntity .
+
+:IdentifierCharacteristic a samm:Characteristic ;
+   samm:preferredName "Identifier Characteristic"@en ;
```
Comment (agg3fe) [inline io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl:335]: `description` missing

File: io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl [hunk 2]
```diff
@@ -0,0 +1,580 @@
...
+
+:continuousRegeneratedPower a samm:Property ;
+   samm:preferredName "Continuous Regenerated Power"@en ;
+   samm:description "The amount of power, which can be regenerated continuously, within the bounds of normal operation, in kilowatts."@en ;
+   samm:characteristic :KilowattMeasurement ;
+   samm:exampleValue "5.0"^^xsd:float .
+
+:maximumRegeneratedPower a samm:Property ;
+   samm:preferredName "Maximum Regenerated Power"@en ;
+   samm:description "The maximum possible value that can be regenereated at a given time in kilowatts."@en ;
```
Comment (agg3fe) [inline io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl:474]: typo `regenereated`

File: io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl [hunk 3]
```diff
@@ -0,0 +1,580 @@
...
+   samm:description "Entity for cooling and lubricants."@en ;
+   samm:dataType :LiquidsEntity .
+
+:InverterCharacteristic a samm:Characteristic ;
+   samm:preferredName "Inverter Characteristic"@en ;
+   samm:description "Characteristic for inverter values."@en ;
+   samm:dataType :InverterEntity .
+
+:SystemCharacteristic a samm:Characteristic ;
+   samm:preferredName "SystemCharacteristic"@en ;
```
Comment (agg3fe) [inline io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl:126]: space required in `SystemCharacteristic`

File: io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl [hunk 4]
```diff
@@ -0,0 +1,580 @@
...
+   samm:properties ( :generalInformation :eMachine :coolingAndLubricants :inverter :systemAndPerformance :transmission :electricDriveDocuments ) .
+
+:UnspecificEntity a samm:Entity ;
+   samm:preferredName "Unspecific Entity"@en ;
+   samm:description "Entity for the product unspecific parameters."@en ;
+   samm:properties ( ext-passport:metadata ext-passport:identification ext-passport:operation ext-passport:handling ext-passport:characteristics ext-passport:commercial ext-passport:materials ext-passport:sustainability ext-passport:sources ) .
+
+:generalInformation a samm:Property ;
+   samm:preferredName "General Information"@en ;
+   samm:description "General information to the eDrive system, including additional information in comparision to the identification category such as the service history and compatible vehicle types."@en ;
```
Comment (agg3fe) [inline io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl:72]: typo `comparision`

File: io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl [hunk 5]
```diff
@@ -0,0 +1,580 @@
...
+
+:oilModule a samm:Property ;
+   samm:preferredName "Oil Module"@en ;
+   samm:description "Flag indicating the availability or absence of an oil module. An oil module is an umbrella term for devices that are manipulating the oil flow and the temperature in the product and incorporate cooler, pump, valves (pressure regulation valve)."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:maximumOilTemperature a samm:Property ;
+   samm:preferredName "Maximum Oil Temperature"@en ;
+   samm:description "The maximum oil temperature to safely operate the eDrive, measured in celsius on the oil temperature sensor. Please note the oil temperatur can vary in other positions within the lubrication circuit."@en ;
```
Comment (agg3fe) [inline io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl:257]: typo `temperatur`

File: io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl [hunk 6]
```diff
@@ -0,0 +1,580 @@
...
+   samm:dataType :TorqueEntity .
+
+:StringList a samm-c:List ;
+   samm:preferredName "String List"@en ;
+   samm:description "A list of values in the data type string."@en ;
+   samm:dataType xsd:string .
+
+:VehicleTypesConstraint a samm-c:RegularExpressionConstraint ;
+   samm:preferredName "Vehicle Types Constraint"@en ;
+   samm:description "Constraint for the three possible vehicle types. A selction of multiple is possible."@en ;
```
Comment (agg3fe) [inline io.catenax.electric_drive.electric_drive_passport/1.0.0/ElectricDrivePassport.ttl:408]: typo `selction`

**Review by agg3fe — CHANGES_REQUESTED** [review]
please check my comments

**Comment by catroest** [comment]
> please check my comments

corrected typos, I missed my spelling check at the end. Now all problems should be solved.

**Review by agg3fe — APPROVED** [review]
LGTM

**Pull request merged**